bin/population/pycbc_population_plots

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python
+
+# Copyright (C) 2017 Vaibhav Tiwari
+
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+"""
+This script reads the rates priors and posteriors and plots them
+"""
+
+__author__ = "Vaibhav Tiwari"
+__email__ = "vaibhav.tiwari@ligo.org"
+__version__ = "0.0"
+__date__ = "31.10.2017"
+
+import logging
+import argparse
+import pycbc.version
+
+import h5py
+import pylab
+import numpy as np
+import scipy.stats as ss
+from matplotlib.pyplot import cm
+from pycbc.population import rates_functions as rf
+from numpy import logaddexp, log, newaxis, expm1
+
+# Parse command line
+parser = argparse.ArgumentParser(description = 
+                                      "Plot rate prior and posteriors.")
+parser.add_argument('--prior-samples', dest='prior_samples', required = True, 
+                  help="Samples for rate prior.")
+parser.add_argument('--posterior-samples', dest='post_samples', nargs = '+',
+                  required = True, help="Samples for rate posterior.")
+parser.add_argument('--population-models', dest='pop_models', nargs = '+',
+                  required=True, help="The corresponding population models.")
+parser.add_argument('--output-rates', dest='rate_file', required=True,
+                  help="File saving all the rate posteriors into one.")
+parser.add_argument('--rates-figure', dest='rate_fig', required=True,
+                  help="Name of file to draw rates prior and posterior.")
+parser.add_argument('--pastro-figure', dest='pastro_fig', required=True,
+                  help="Name of file to save p_astro figure.")
+parser.add_argument('--plot-labels', dest='plot_labels', nargs = '+',
+                  required = True, help="Labels for the population models.")
+
+opts = parser.parse_args()   
+#Save rate posteriors in one file
+with h5py.File(opts.rate_file, 'w') as out:
+    for fname, model in zip(opts.post_samples, opts.pop_models):
```

Comment:
Before reaching this point, should do an input check that `opts.post_samples` and `opts.pop_models` have the same number of elements.  I.e. `assert len(opts.post_samples) = len(opts.pop_models)`

Reply:
asserting now. Thanks